### ISSUE TITLE
homebridge-lifx is not in the npm registry

### ISSUE THREAD

**Issue by tokamac** [madpilot/homebridge-lifx#1]
Hi,
You wrote: "Install this plugin using: npm install -g homebridge-lifx"
Actually your LIFX plugin is not available on the npm website:
https://www.npmjs.com/browse/keyword/homebridge-plugin

And
`npm install homebridge-lifx`
returns:
`npm ERR! code E404
npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/homebridge-lifx
npm ERR! 404 
npm ERR! 404 'homebridge-lifx' is not in the npm registry.`

**Comment by madpilot** [comment]
Yeah, as I didn't write this plugin, I just copied it out of the legacy plugin, I didn't want to take ownership on NPM.

You can check out the repo and do:

npm install -g ./ from the repo directory and install it that way

**Comment by tokamac** [comment]
Ok, so do you know who wrote it? I want to ask the dev if he can port his shim to the new npm plugin architecture. But I don't find credits in actual LIFX-related files.

**Comment by madpilot** [comment]
Not really. Using blame on github if looks like https://github.com/nmelo

**Comment by tokamac** [comment]
Thanks.
